DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -664,13 +652,26 @@ namespace pat {
     reco::VertexRef::key_type pvRefKey_;
 
     /// IP covariance	
-    CMS_THREAD_GUARD(vertex_) mutable float dxydxy_, dzdz_, dxydz_,dlambdadz_,dphidxy_,dptdpt_,detadeta_,dphidphi_;
+    CMS_THREAD_GUARD(vertex_) mutable reco::TrackBase::CovarianceMatrix m_;
```

Comment:
I'm not sure I understood how you would use the atomic pointer, I mean the problem is that the assignment to m_  is a several steps operation. Would you suggest to have a temporary object to work on (passing it to the various functions) and then assign to the datamember with .store() once finished?  I think I'm a bit lost, if I've to pass it along, then I could consider just removing m_ and keeping it only as a datamember of the mutable track_ (it is already there in fact and the only advantage of having it as a member was to avoid passing the object around)